.travis.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/env bash
```

Comment:
@Bakley, delete this file from the commit. It is no longer needed.

Reply:
Deleted